service/gcs/bridge/bridge.go

```diff
@@ -191,6 +193,7 @@ func (b *bridge) createContainer(message []byte) (*prot.ContainerCreateResponse,
 	// CreateContainerInfo struct.
 	var settings prot.VMHostedContainerSettings
 	if err := json.Unmarshal([]byte(request.ContainerConfig), &settings); err != nil {
+		err = gcserr.WrapHresult(err, gcserr.HrVmcomputeInvalidJSON)
```

Comment:
Maybe we need a helper method (unmarshalProtocolJson?) that does an unmarshal and wraps the error with this if it fails.